### PR TITLE
A mixed bag

### PR DESCRIPTION
**Pull request by theseal**
* A new class to handle acme-d througt certbot. Will replace https://wiki.sunet.se/display/sunetops/How+to+create+a+cert+with+acme-d
* A new class (WIP) to setup get eduroam
* A few other tweaks…

### CODE REVIEW

File: files/certbot/certbot-acmed-renew-post-hook-wrapper
Comment (eest) [inline files/certbot/certbot-acmed-renew-post-hook-wrapper:1]: Would we want to store this in a separate dir, something like /etc/sunet-certbot-acmed instead? Just wondering if it is OK to do your own thing under /etc/letsencrypt or not.

Comment (theseal) [reply files/certbot/certbot-acmed-renew-post-hook-wrapper:1]: Problem solved by reading the docs and getting help from mifr. We new uses certbots own hook dir(s).

**Review by eest — APPROVED** [review]
LGTM

**Review by mikaelfrykholm — APPROVED** [review]
LGTM

**Pull request merged**